test/lib/rubycritic/analysers/smells/reek_test.rb

```diff
@@ -11,25 +11,15 @@
     end
 
     it "detects its smells" do
-      @analysed_module.smells.length.must_equal 1
+      @analysed_module.smells.length.must_equal 2
     end
 
     it "creates smells with messages" do
-      smell = @analysed_module.smells.first
-      smell.message.must_equal "has boolean parameter 'reek'"
-    end
-  end
-
-  context "when analysing a file with smells ignored in config.reek" do
```

Comment:
This context shouldn't be here anymore since handling it's configuration properly is now `Reek`'s task.

I do believe we should test this though, but then not as a unit test but rather a full blown integration test which would be out of scope for this PR - i'll look into setting up an integration test suite separetely.